test/fuzzer/pedro/another_binder_error.test

```diff
@@ -0,0 +1,25 @@
+# name: test/fuzzer/pedro/another_binder_error.test
+# description: Issue #4978 (issue 48): Another Binder Issue
+# group: [pedro]
+
+statement error
+SELECT avg(0) c0, (SELECT 0 OFFSET c0);
+----
+Parser Error: Non-constant limit or offset not supported in correlated subquery
+
+statement error
+SELECT avg(0) c0, (SELECT 0 OFFSET c0 + 1);
+----
+Parser Error: Non-constant limit or offset not supported in correlated subquery
```

Comment:
The same error happens if we leave out the offset, e.g.:

```sql
D select avg(0) AS c0, (SELECT c0);
Error: INTERNAL Error: Failed to bind column reference "c0" [2.0] (bindings: [15.0])
```

Could you test that that is fixed as well?

Reply:
@Mytherin added a test case for this, it's fixed as well!